specs/CONTROL.md

```diff
@@ -21,8 +21,7 @@ There are two pieces to the libp2p daemon:
 
 ### Technical Details
 
-The libp2p daemon and client will communicate with each other over a simple Unix
-socket based protocol built with [protobuf](https://developers.google.com/protocol-buffers/).
+The libp2p daemon and client will communicate with each other over any multiaddr supported protocol with [protobuf](https://developers.google.com/protocol-buffers/).
```

Comment:
again here, developer don't care about the multiaddr detail... 
"they communicate over a simple socket based protocol" is sufficient.